### PR TITLE
Fixes an oversight allowing players to list ckeys and the names of the mobs they're controlling under certain conditions

### PR DESCRIPTION
**Pull request by KathrinBailey**
## About The Pull Request
• Resolves a probably unknown oversight allowing players to find out who plays who by using the ignore verb, which if the user is a ghost, will list the names of people's mobs alongside their ckey if they are also a ghost.

## Why It's Good For The Game
Removes a potential method of metagrudging and addresses a privacy concern for those who wish to play anonymous characters. _Yes, you can still see this stuff in the round-end report._
We don't lose anything here. If we're using this verb, we're doing it because we want to ignore someone's OOC messages. We don't need this additional information. I feel like this is just an oversight.

## Changelog
:cl:
fix: Players can no longer us the ignore verb to gather a list of people's characters and ckeys together.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
have you _looked_ at that code youre changing

**Comment by Tlaltecuhtli** [comment]
privacy, with public logs...................

**Comment by optimumtact** [comment]
You're both conveniently ignoring that those are only available post round

File: code/modules/client/verbs/ooc.dm [hunk 1]
```diff
@@ -254,9 +254,10 @@ GLOBAL_VAR_INIT(normal_ooc_colour, "#002eb8")
 
 		// Check if both we and the player are ghosts and they're not using a fakekey
 		if(isobserver(mob) && isobserver(C.mob) && !C.holder?.fakekey)
-			// Show us the player's mob name in the list in front of their displayed key
-			// Add the player's displayed key to the list
-			players["[C.mob]([displayed_key])"] = displayed_key
+			// We used to look for the player's mob name and display it alongisde their ckey using the line below.
+			//players["[C.mob]([displayed_key])"] = displayed_key
```
Comment (optimumtact) [inline code/modules/client/verbs/ooc.dm:258]: just delete it, don't comment it out, git will take care of the history

**Comment by ArcaneDefence** [comment]
>If we're using this verb, we're doing it because we want to ignore someone's OOC messages

Does it block their IC messages too?

File: code/modules/client/verbs/ooc.dm (hunk 1 above)
Comment (KathrinBailey) [reply code/modules/client/verbs/ooc.dm]: Done

**Comment by Fikou** [comment]
> You're both conveniently ignoring that those are only available post round

how do u metagrudge someone if both of you are dead ghosts

also this pr STILL does this
if(x)
     y
else
     y

File: code/modules/client/verbs/ooc.dm [hunk 2]
```diff
@@ -254,9 +254,7 @@ GLOBAL_VAR_INIT(normal_ooc_colour, "#002eb8")
 
 		// Check if both we and the player are ghosts and they're not using a fakekey
 		if(isobserver(mob) && isobserver(C.mob) && !C.holder?.fakekey)
```
Comment (Cyprex) [inline code/modules/client/verbs/ooc.dm:256]: You should remove this if and the else block below.

**Review by Cyprex — CHANGES_REQUESTED** [review]

**Comment by wesoda25** [comment]
cringe

**Comment by wesoda25** [comment]
Also how are you supposed to ignore ghosts now? It will just display their ckey so you won't know who it is. It should display their ghost name with (ghost) next to it instead of ckey(ghost).

**Comment by optimumtact** [comment]
close the game

**Pull request merged**